### PR TITLE
Extending Websockets in UIPublisher to allow query-strings in connect…

### PR DESCRIPTION
**Pull request by Shabirmean**
…ions

### CODE REVIEW

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/internal/util/CEPWebSocketSession.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package org.wso2.carbon.event.output.adapter.ui.internal.util;
```
Comment (keizer619) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/internal/util/CEPWebSocketSession.java]: @Shabirmean  Include License header

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/internal/ds/OutputAdaptorEventStreamServiceValueHolder.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package org.wso2.carbon.event.output.adapter.ui.internal.ds;
```
Comment (keizer619) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/internal/ds/OutputAdaptorEventStreamServiceValueHolder.java]: @Shabirmean  Include License header

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/internal/util/CEPWebSocketSession.java (hunk 1 above)
Comment (Shabirmean) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/internal/util/CEPWebSocketSession.java]: @keizer619 Added bro

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/internal/ds/OutputAdaptorEventStreamServiceValueHolder.java (hunk 2 above)
Comment (Shabirmean) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/internal/ds/OutputAdaptorEventStreamServiceValueHolder.java]: @keizer619 Added bro

**Comment by Shabirmean** [comment]
Made suggested changes. Please note that the PR is for the release 5.0.10 branch.
I believe I have to do the same to the master as well

**Comment by dilini-muthumala** [comment]
@Shabirmean Hi, Could you please send this PR for the master branch?

**Comment by Shabirmean** [comment]
@dilini-muthumala Sent the PR to master. Please check and verify.

**Comment by mohanvive** [comment]
This PR is no longer valid hence closing it.

**Pull request closed without merging**